tuf/client_rework/updater_rework.py

```diff
@@ -610,10 +424,8 @@ def _preorder_depth_first_walk(self, target_filepath) -> Dict:
             and len(role_names) > 0
         ):
             msg = (
-                f"{len(role_names)}  roles left to visit, ",
-                "but allowed to visit at most ",
-                f"{settings.MAX_NUMBER_OF_DELEGATIONS}",
-                " delegations.",
+                f"{len(role_names)} roles left to visit, but allowed to ",
+                f"visit at most {MAX_DELEGATIONS} delegations.",
             )
             logger.debug(msg)
```

Comment:
This trick of mine with logging needs to be reverted now that we agreed on %s-strings in the logging calls. 
But not in this PR. I'll note it for myself.

Reply:
Captured in #1400